<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWatchDialog</class>
 <widget class="QMainWindow" name="MainWatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>544</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_setting">
       <attribute name="title">
        <string>监视设置</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>模式</string>
        </property>
        <widget class="QRadioButton" name="radioButton_inj">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>注入模式</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_dbg">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>调试模式</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>181</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>监视模块</string>
        </property>
        <widget class="QCheckBox" name="checkBox_lvl1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>79</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Exe文件</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_lvl2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Exe路径下DLL</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_lvl3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>其他路径模块</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_lvl4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>79</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>系统模块</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>监视来源</string>
        </property>
        <widget class="QCheckBox" name="checkBox_Iat">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>导入表</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_Gpa">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>GetProcAddress</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>241</width>
          <height>291</height>
         </rect>
        </property>
        <property name="title">
         <string>数据提取设置</string>
        </property>
        <widget class="QComboBox" name="comboBox_DataType">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>类型</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>过滤条件</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_FilterType">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>参数</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_ExtraParam1">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>100</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_ExtraParam2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>100</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>排序方式</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_SortType">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>140</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>170</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>参数</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_ExtraParam">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>170</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>最大显示数</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_MaxCount">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>210</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_ApplyDataPick">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>250</y>
           <width>75</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>应用</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_Undo">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>350</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>撤销</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Install">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>350</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>安装</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_show">
       <attribute name="title">
        <string>数据显示</string>
       </attribute>
       <widget class="QRadioButton" name="radioButton_loop">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>320</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>启用循环</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_refresh">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>350</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>函数调用记录</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>41</y>
          <width>441</width>
          <height>241</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QTextEdit" name="textEdit_MoreInfo">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>291</width>
          <height>71</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_cmd">
       <attribute name="title">
        <string>高级</string>
       </attribute>
       <widget class="QTextEdit" name="textEdit_reply">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>341</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cmd">
        <property name="geometry">
         <rect>
          <x>122</x>
          <y>360</y>
          <width>341</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_help">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>帮助</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>360</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>指令：</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="separator"/>
   <addaction name="actionRestart"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionDetach"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="FuncWatcher.qrc">
     <normaloff>:/FuncWatcher/MyIcons/FileIcon.png</normaloff>:/FuncWatcher/MyIcons/FileIcon.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>选择一个可执行文件</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="FuncWatcher.qrc">
     <normaloff>:/FuncWatcher/MyIcons/StopIcon.png</normaloff>:/FuncWatcher/MyIcons/StopIcon.png</iconset>
   </property>
   <property name="text">
    <string>终止</string>
   </property>
   <property name="toolTip">
    <string>终止被监视进程</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="FuncWatcher.qrc">
     <normaloff>:/FuncWatcher/MyIcons/RestartIcon.png</normaloff>:/FuncWatcher/MyIcons/RestartIcon.png</iconset>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
   <property name="toolTip">
    <string>重启进程</string>
   </property>
  </action>
  <action name="actionDetach">
   <property name="icon">
    <iconset resource="FuncWatcher.qrc">
     <normaloff>:/FuncWatcher/MyIcons/DetachIcon.png</normaloff>:/FuncWatcher/MyIcons/DetachIcon.png</iconset>
   </property>
   <property name="text">
    <string>脱离</string>
   </property>
   <property name="toolTip">
    <string>断开调试连接</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="FuncWatcher.qrc">
     <normaloff>:/FuncWatcher/MyIcons/StartIcon.png</normaloff>:/FuncWatcher/MyIcons/StartIcon.png</iconset>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
   <property name="toolTip">
    <string>创建进程</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FuncWatcher.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Install</sender>
   <signal>clicked()</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>InstallAndStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Undo</sender>
   <signal>clicked()</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>UninstallHooks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_FilterType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>ChangeFilterType(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_SortType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>ChangeSortType(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_loop</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>SetLoopMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>RefreshData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>GetMoreInfo(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_help</sender>
   <signal>clicked()</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>ShowHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_cmd</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>ExecuteCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_inj</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>SetInjectMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_dbg</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>SetDebugMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ApplyDataPick</sender>
   <signal>clicked()</signal>
   <receiver>MainWatchDialog</receiver>
   <slot>ApplyDataPickSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>InstallAndStart()</slot>
  <slot>UninstallHooks()</slot>
  <slot>ChangeFilterType(int)</slot>
  <slot>ChangeSortType(int)</slot>
  <slot>SetLoopMode(bool)</slot>
  <slot>RefreshData()</slot>
  <slot>GetMoreInfo(int,int)</slot>
  <slot>ShowHelp()</slot>
  <slot>ExecuteCommand()</slot>
  <slot>SetInjectMode(bool)</slot>
  <slot>SetDebugMode(bool)</slot>
  <slot>ApplyDataPickSettings()</slot>
 </slots>
</ui>
